lang/es.po

```diff
@@ -40,7 +41,11 @@ msgstr "    Controla si la terminal debería de usar sequencias de escape\n"
        "    redirecciona a un archivo o a un pipe.\n"
 
 msgid "HELP_OPTION_DESCRIBE\n"
-msgstr "    Cuando enlistas paquetes, incluye descripción corta de los mismos.\n"
+msgstr "    Al listar paquetes, incluya los resúmenes de paquetes (descripciones breves).\n"
+
+msgid "HELP_OPTION_EVRA\n"
+msgstr "    Además de los nombres de los paquetes, imprima también su epoch:version-release.arch.\n"
```

Comment:
Please wrap this to 80 characters per line - that being not 80 characters per line of the `.po` file, but 80 characters per line of the message (so from the opening `"` to the closing `\n"`).

Note: Wrapping is only needed for the `HELP_*` strings. `ERR_*` strings should be left single-line.